deploy/charts/arcadia/templates/pg-init-data-configmap.yaml

```diff
@@ -326,7 +326,50 @@ data:
         COMMENT ON COLUMN public.data_process_task_stage_log.update_datetime IS '更新时间';
         COMMENT ON COLUMN public.data_process_task_stage_log.update_user IS '更新人';
         COMMENT ON COLUMN public.data_process_task_stage_log.update_program IS '更新程序';
+        
+        CREATE TABLE IF NOT EXISTS data_process_task_document_web_url
```

Comment:
@GHOSTHSP @wangxinbiao  Please note that we should alway upgrade the chart version once you update it

Reply:
@GHOSTHSP  upgrade the chart version. deploy/charts/arcadia/Chart.yaml

Reply:
The version has been updated to 0.1.29